src/Simplex/Chat/Library/Commands.hs

```diff
@@ -4163,6 +4171,39 @@ agentSubscriber = do
 
 type AgentSubResult = Map ConnId (Either AgentErrorType (Maybe ClientServiceId))
 
+runRelationsVectorMigration :: CM ()
+runRelationsVectorMigration = do
+  liftIO $ threadDelay' 5000000 -- 5 seconds (initial delay)
+  migrateUserNotAdminGroups
+  migrateMembers
+  where
+    stepDelay = 100000 -- 0.1 second
+    migrateUserNotAdminGroups = flip catchAllErrors eToView $ do
+      lift waitChatStartedAndActivated
+      gIds <- withStore' getUserNotAdminGroupIds
+      forM_ gIds $ \gId -> do
+        withStore' (\db -> setEmptyVectorForGroupId db gId) `catchAllErrors` eToView
```

Comment:
you don't need to read them, can be just single update query.

Reply:
also, this can be done as part of schema migration, so it doesn't need to be code.